### PR TITLE
Create log in

### PR DESCRIPTION
**Pull request by hoadt98**
==> Tittle: Create log in

==> Redmine: https://edu-redmine.sun-asterisk.vn/projects/book-reviewing

==> Attachment File: none.

### CODE REVIEW

**Comment by hoadt98** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -1,11 +1,14 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
```
Comment (phucmh-1356) [inline config/routes.rb]: dùng " thay cho ', cho vào cụm get đi em

**Comment by hoadt98** [comment]
ready

**Comment by phucmh-1356** [comment]
commented
gộp commits đi em

**Comment by hoadt98** [comment]
ready

**Pull request merged**